tree/ntuple/v7/src/RClusterPool.cxx

```diff
@@ -115,7 +116,8 @@ void ROOT::Experimental::Detail::RClusterPool::ExecReadClusters()
          while (!fReadQueue.empty()) {
             if (fReadQueue.front().fClusterKey.fClusterId == kInvalidDescriptorId) {
                fReadQueue.pop();
-               return;
+               terminate = true;
+               break;
```

Comment:
In this error case, should we still call "LoadClusters" on the others clusters or should we just discard them all?

Reply:
We should still call `LoadClusters` on the ones that were in the pipe before the termination marker. @eguiraud pointed the issue out from the [unit tests](https://lcgapp-services.cern.ch/root-jenkins/job/roottest-pullrequests-build/9815/testReport/projectroot.tree.ntuple.v7/test/gtest_tree_ntuple_v7_test_ntuple_cluster)

Reply:
On second thought, I think actually we shouldn't.  Why would we need to continue prefetching if the cluster pool gets destructed.  That makes testing a bit more complicated, I need to give it one more round of thought.  Thanks for the comment!